### ISSUE TITLE
Enabling default layer does not show technique matrix

### ISSUE THREAD

**Issue by WheelsVT** [mitre/attack-navigator#59]
When I save and reference a default layer in the config.json file or by trying to make a custom URL referencing a hosted layer file the layer loads, however, the technique matrix does not show up. All I see is the layer title and label while the matrix itself never loads.

Perhaps user error?

**Comment by isaisabel** [comment]
Hi WheelsVT,
1) Could you send us a picture of your javascript console after the error occurs? You can open the console in google chrome with view -> developer -> javascript console. 

2) Could you try pasting `https://raw.githubusercontent.com/mitre/attack-navigator/master/layers/data/samples/Bear_APT.json` in the "default layer url" box in the "create customized navigator" screen? <img width="1132" alt="screen shot 2018-09-19 at 2 31 38 pm" src="https://user-images.githubusercontent.com/28868888/45773710-fc201c80-bc18-11e8-85bc-2eb92600543e.png"> Then copy the output below and see if that works. 

Thanks! 

Isabel

**Comment by christophetd** [comment]
I'm having a similar issue. Importing the Bear APT layer does not change anything. I tried running the app both in Docker and directly using `npm install && ng serve`. Here's my console output:

![image](https://user-images.githubusercontent.com/136675/45917976-3551ca00-be6f-11e8-96cc-3b5e664e9bab.png)

UI screenshot:

![image](https://user-images.githubusercontent.com/136675/45917980-439fe600-be6f-11e8-9ff9-a94961161325.png)

Tested with latest Chrome and Firefox
